app/(auth)/sign-in/index.tsx

```diff
@@ -34,21 +34,21 @@ const Page = () => {
   const [selectedCountry, setSelectedCountry] = useUsedCountryStorage()
 
   useEffect(() => {
-    if (selectedCountry) return
-
-    setSelectedCountry('SK')
+    if (!selectedCountry) {
```

Comment:
why don't we use first approach? I think it's more readable when used consistently like that